File: Solution/Java/SarthakW/QuestionBank/src/main/java/com/tap/sarthak/Entity/Question.java
package com.tap.sarthak.Entity;

import java.lang.invoke.StringConcatException;

public class Question {

    private String title;
    private int id;
    private int subjectId;
    private String optionA;
    private String optionB;
    private String optionC;
    private String optionD;
    private String CorrectAnswer;
    private int EvaluationCritera;

    public Question() {

    }

    public Question(String title, String optionA, String optionB, String optionC, String optionD, String CorrectAnswer,
            int EvaluationCritera, int id, int subjectId) {
        this.title = title;
        this.optionA = optionA;
        this.optionB = optionB;
        this.optionC = optionC;
        this.optionD = optionD;
        this.CorrectAnswer = CorrectAnswer;
        this.EvaluationCritera = EvaluationCritera;
        this.id = id;
        this.subjectId = subjectId;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getSubjectId() {
        return subjectId;
    }

    public void setSubjectId(int subjectId) {
        this.subjectId = subjectId;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getOptionA() {
        return optionA;
    }

    public void settOptionA(String optionA) {
        this.optionA = optionA;
    }

    public String getOptionB() {
        return optionB;
    }

    public void setOptionB(String optionB) {
        this.optionB = optionB;
    }

    public String getOptionC() {
        return optionC;
    }

    public void setOptionC(String optionC) {
        this.optionC = optionC;
    }

    public String getOptionD() {
        return optionD;
    }

    public void setOptionD(String optionD) {
        this.optionD = optionD;
    }

    public String getCorrectAnswer() {
        return CorrectAnswer;
    }

    public void setCorrectAAnswer(String CorrectAnswer) {
        this.CorrectAnswer = CorrectAnswer;
    }

    public int getEvaluationCritera() {
        return EvaluationCritera;
    }

    public void setEvaluationCritera(int EvaluationCritera)
    {
        this.EvaluationCritera = EvaluationCritera;
    }

    @Override
    public String toString() {
        return "\n\nQuestion{" + "title=" + title + ", id=" + id + ", optionA=" + optionA +
                ", optionB=" + optionB +
                ", optionC=" + optionC +
                ", optionD=" + optionD +
                ", correctAnswer=" + CorrectAnswer +
                ", subject Id=" + subjectId +
                ", evaluationCriteria=" + EvaluationCritera +
                "}";
    }
}

File: Solution/Backend/dotnet/DataHelperClasslib_Generic/CSVFileIo/Entities/Question.cs
using System.Dynamic;

namespace CSVFileIO.Entity{
public class Question
{
    public int Id{get;set;}
    public string Concept{get;set;}
    public string Subject{get;set;}
    public string Title{get;set;}
    public string A{get;set;}
    public string B{get;set;}
    public string C{get;set;}
    public string D{get;set;}
    public string AnswerKey{get;set;}
    public string DifficultyLevel{get;set;}
    public string CreatedBy{get;set;}
  
    public Question()
        {
            
        }
    public Question(int id, string concept, string subject,string title,string a,string b, string c, string d, string answerKey,string difficultyLevel,string createdBy)
    {
        this.Id = id;
        this.Concept = concept;
        this.Subject = subject;
        this.Title = title;
        this.A = a;
        this.B = b;
        this.C = c;
        this.D = d;
        this.AnswerKey = answerKey;
        this.DifficultyLevel = difficultyLevel;
        this.CreatedBy = createdBy;
    }

    
    public  string ToString()
        {
            return Id + "," + Concept + "," + Subject + "," + "\"" + Title + "\"," + A + "," + B + "," + C + "," + D + "," + AnswerKey + "," + DifficultyLevel + "," + CreatedBy;
        }
}
}